bigtable/Dockerfile

```diff
@@ -0,0 +1,22 @@
+## Build
+FROM golang:1.11-buster AS build
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY bttest/ ./bttest
+COPY cmd/ ./cmd
+
+RUN go mod download
+RUN cd cmd/cbtemulator/ && go build -o ../../cbtemulator
+
+## Deploy
+FROM gcr.io/distroless/base-debian10
+
+WORKDIR /
+
+COPY --from=build /app/cbtemulator /cbtemulator
+
+EXPOSE 9000
+
+ENTRYPOINT ["/cbtemulator", "-port", "9000", "-host", "0.0.0.0", "-dir", "var/bigtable"]
```

Comment:
I assume you can map internal port 9000 to external port whatever as part of the docker spin up?

Reply:
yeah, we could run the container using docker run --rm -p 8080:9000 -it cbtemulator for example. 

Using testcontainers is a little bit diff cause it will map the internal port, but the exposed one will only be given on runtime (it gives a random port to avoid conflicts)